changelog/fix_github_page_load_errors_for_large_changelogs_20250408151025.md

```diff
@@ -0,0 +1 @@
+* [#14085](https://github.com/rubocop/rubocop/pull/14085): Fix Github page load errors for large markdown file. ([@corsonknowles][])
```

Comment:
Ideally, the page corresponding to the CHANGELOG should render properly on GitHub. Since this is not an issue with RuboCop itself, a changelog entry is unnecessary.